aggregator/src/lib.rs

```diff
@@ -6,3 +6,9 @@ pub mod binary_utils;
 pub mod config;
 pub mod metrics;
 pub mod trace;
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+enum Operation {
```

Comment:
That's where I originally stuck it, but it turned out to also be useful in `aggregator.rs` (specifically, in `handle_create_collection_generic`, where it is used to track whether batches to be written are new or pre-existing).

Since I use it throughout the aggregator codebase, but it's otherwise a pretty generic data structure, I put it in `lib.rs`. Thoughts on a better location if we want to go more specific?

Reply:
Maybe `aggregator_core::datastore`? But I suppose it doesn't really matter, so let's leave it here in `lib.rs`.